src/frontend/src/utils/enum-pipes.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { yellow, green, blue, purple, grey } from '@mui/material/colors';
+import { WorkPackageStage } from 'shared';
+
+// maps stage to the desired color
+export function WorkPackageStageColorPipe(stage: WorkPackageStage | undefined): string {
```

Comment:
we dont really use function syntax we say export const WorkPacka... = (stage: Wo...) => {
...
}